src/os/bluestore/BlueFS.cc

```diff
@@ -3118,15 +3129,15 @@ int BlueFS::_allocate(uint8_t id, uint64_t len,
     }
     if (id != BDEV_SLOW) {
       if (bdev[id]) {
-	dout(1) << __func__ << " failed to allocate 0x" << std::hex << len
+	dout(3) << __func__ << " failed to allocate 0x" << std::hex << len
```

Comment:
since you're touching this line - could you please replace "failed" with "unable"? IMO it'll better reflect the status and sounds less critical.